crnk-core/src/main/java/io/crnk/core/module/ModuleRegistry.java

```diff
@@ -97,6 +87,8 @@
 
 	private ResourceFilterDirectory filterBehaviorProvider;
 
+	private List<PagingBehavior> pagingBehaviors = new ArrayList<>();
```

Comment:
potentially does is not necessary, since one can add new paging behaviors through a module